index.js

```diff
@@ -33,19 +33,29 @@ expressApp.listen(process.env.PORT || 8080);
 /**
  * Default Welcome Intent
  */
-app.intent('Default Welcome Intent', conv => {
+app.intent('Default Welcome Intent', async conv => {
   // Create a session for this user at the beginning so user
   // is logged on to Circuit by the time needed
   if (!sessions[conv.user.id]) {
     createSession(conv.user);
   }
 
+  let { inConferenceCall, conferenceAvailable } = await getConferenceInfo(conv);
   conv.ask(`What can I do for you?`);
-  conv.ask(new Suggestions('Send a message', 'Make a call'));
+ 
```

Comment:
Create an array here for the suggestions with Send a message', 'Make a call'. Then just add yours in the if statements below to the array, and the "ask" at the end. This avoid duplicate code adding the suggestions that are always used.